### PR TITLE
feat: Reusable credential detail UI component layout

### LINKED ISSUE #1008: Credential detail ui component

### PR DESCRIPTION
**Pull request by talwinder50**
closes #1008 

Once the integration is done below will be layout. This layout is with static data input 

Screen Shot 2021-08-30 at 1 21 14 AM

### CODE REVIEW

File: cmd/wallet-web/src/assets/img/Icons _ sm--more-icon.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+version https://git-lfs.github.com/spec/v1
```
Comment (birtony) [inline cmd/wallet-web/src/assets/img/Icons _ sm--more-icon.svg:1]: We don't need this file

File: cmd/wallet-web/src/pages/DIDManagement.vue [hunk 2]
```diff
@@ -7,9 +7,9 @@
 <template>
   <div class="flex flex-col justify-start items-start w-full">
     <tabs>
-      <tab title="Digital Identity Preference"><preferences :allDIDs.sync="allDIDs" /></tab>
-      <tab title="Create ORB Digital Identity"><create-orb :allDIDs.sync="allDIDs" /></tab>
-      <tab title="Import Any Digital Identity"><import-did :allDIDs.sync="allDIDs" /></tab>
+      <tab title="Digital Identity Preference"><preferences :all-d-i-ds.sync="allDIDs" /></tab>
+      <tab title="Create ORB Digital Identity"><create-orb :all-d-i-ds.sync="allDIDs" /></tab>
+      <tab title="Import Any Digital Identity"><import-did :all-d-i-ds.sync="allDIDs" /></tab>
```
Comment (birtony) [inline cmd/wallet-web/src/pages/DIDManagement.vue:12]: Eh...we need to come up with some other naming convention to avoid this stuff...

**Review by birtony — COMMENTED** [review]
Let's remove that extra icon and merge, the rest can be addressed in follow-up PRs

**Comment by talwinder50** [comment]
> Let's remove that extra icon and merge, the rest can be addressed in follow-up PRs

done

**Review by birtony — APPROVED** [review]

File: cmd/wallet-web/src/pages/DIDManagement.vue (hunk 2 above)
Comment (birtony) [reply cmd/wallet-web/src/pages/DIDManagement.vue:12]: Let's make sure this does not break the DID management page @talwinder50

**Review by rolsonquadras — APPROVED** [review]

**Pull request merged**